src/displayapp/screens/WatchFaceDigital.h

```diff
@@ -8,6 +8,7 @@
 #include "displayapp/screens/Screen.h"
 #include "components/datetime/DateTimeController.h"
 #include "components/ble/BleController.h"
+#include "components/alarm/AlarmController.h"
```

Comment:
reordering includes to be alphabetically

```suggestion
#include "components/alarm/AlarmController.h"
#include "components/ble/BleController.h"
#include "components/datetime/DateTimeController.h"
```